jobs/JGLOBAL_ATM_ANALYSIS_INITIALIZE

```diff
@@ -27,8 +27,8 @@
     COM_ATMOS_HISTORY_PREV:COM_ATMOS_HISTORY_TMPL \
     COM_ATMOS_RESTART_PREV:COM_ATMOS_RESTART_TMPL
 
-MEMDIR='ensstat' RUN=${GDUMP_ENS} YMD=${gPDY} HH=${gcyc} generate_com -rx \
-    COM_ATMOS_HISTORY_ENS_PREV:COM_ATMOS_HISTORY_TMPL
+MEMDIR='${MEMDIR}' RUN=${GDUMP_ENS} YMD=${gPDY} HH=${gcyc} generate_com -rx \
```

Comment:
If `MEMDIR='${MEMDIR}'` is replaced with `MEMDIR="${MEMDIR}"` or `MEMDIR="'${MEMDIR}'"`, job `atmanlinit` aborts with an `unbound variable` message
```
/scratch1/NCEPDEV/da/role.jedipara/git/global-workflow/test0/jobs/JGLOBAL_ATM_ANALYSIS_INITIALIZE: line 30: MEMDIR: unbound variable
++ JGLOBAL_ATM_ANALYSIS_INITIALIZE[1]: postamble JGLOBAL_ATM_ANALYSIS_INITIALIZE 1692209724 1
++ preamble.sh[68]: set +x
End JGLOBAL_ATM_ANALYSIS_INITIALIZE at 18:15:28 with error code 1 (time elapsed: 00:00:04)
```

How do we pass the shellcheck without breaking existing functionality?

Reply:
I think you want to disable the error since shellcheck does not know that you are intentionally trying to create the template.
Adding a line:
```
shellcheck disable=SC2016
```
 before this line will inform shellcheck to ignore this error.